catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/web/metrics/BlackHoleMetricsStore.java

```diff
@@ -0,0 +1,26 @@
+/*
+ *  Copyright 2023 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics;
+
+import java.time.Instant;
+import java.util.Map;
+import org.apache.iceberg.metrics.MetricsReport;
+
+public class BlackHoleMetricsStore implements IcebergMetricsStore {
+  public static final String ICEBERG_METRICS_STORE_BLACK_HOLE_NAME = "blackHole";
```

Comment:
"blackhole" is a word, using "Blackhole", "BLACKHOLE" and "blackhole".

Reply:
"blackhole" seems not a word, it's "black hole"

Reply:
blackhole is probably a word best to avoid

Reply:
> blackhole is probably a word best to avoid

+1, similar words like blacklists.

Reply:
How about dummyMetricsStore?

Reply:
Spark uses `FirehoseXXX`, how about this?

Reply:
I prefer using `dummy`, `Firehose` seems a unfamiliar word :(

Reply:
Firehose can mean a torrent of data, most native English speakers would understand it meaning